utils/prod/s3_utils.py

```diff
@@ -0,0 +1,172 @@
+import os
+
+import dotenv
+
+dotenv.load_dotenv()
+import io
+import logging
+from typing import BinaryIO, Union
+
+import boto3
+from botocore.exceptions import ClientError
+
+# Setup logging
+logging.basicConfig(level=logging.INFO, format='%(asctime)s - %(levelname)s - %(message)s')
+logger = logging.getLogger(__name__)
+
+
+# Get the S3 bucket name
+BUCKET_AWS_NAME = os.getenv('BUCKET_AWS_NAME')
```

Comment:
should we include a load_dotenv here, or we will always expect the envs to be set when when this is imported?

Reply:
same comment for access keys in methods

Reply:
Good point.

Reply:
It is already there on line 3. I'll move it down for readability.